index.html

```diff
@@ -289,6 +290,37 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact flex" id="contact">
+        <p class="parag text-dark">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          class="flex"
+          action="https://formspree.io/f/xdobralq"
+          method="post"
+          id="mail"
+        >
+          <input
+            name="fname"
+            id="name"
+          />
+          <input
+            name="email"
+            type="email"
+            type="text"
+            required
+            placeholder="Email address"
+            id="email"
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+          ></textarea>
+          <button class="special" type="submit">Get in touch</button>
```

Comment:
- [ ] [ FRIENDLY REMINDER ]  Remember, according to the review policy, you can only submit 3times a PR, then consider making all the necessary required changes before you submit your second review, there you will have more chances to not fall on the last one and maybe result in a block repetition, we don't all need that 😄 

- [ ] As said in the previous review, the button interactions are important for the user experience, this will let the user know when an element is clickable, please make these few changes and resubmit your project.